tools/ci_build/github/linux/docker/scripts/install_onnx.sh

```diff
@@ -1,20 +1,14 @@
 #!/bin/bash
 set -e
 PYTHON_VER=$1
-
-#Install ONNX
-#5af210ca8a1c73aa6bae8754c9346ec54d0a756e is v1.2.3
-#bae6333e149a59a3faa9c4d9c44974373dcf5256 is v1.3.0
-#9e55ace55aad1ada27516038dfbdc66a8a0763db is v1.4.1
-#7d7bc83d29a328233d3e8affa4c4ea8b3e3599ef is v1.5.0
-#5c51f0dbbe88ee1536f17ee7bd462b2ab3772c52 is master
-declare -A version2tag
-version2tag+=(["5af210ca8a1c73aa6bae8754c9346ec54d0a756e"]="onnx123"
-              ["bae6333e149a59a3faa9c4d9c44974373dcf5256"]="onnx130"
-              ["9e55ace55aad1ada27516038dfbdc66a8a0763db"]="onnx141"
-              ["7d7bc83d29a328233d3e8affa4c4ea8b3e3599ef"]="onnx150"
-              ["5c51f0dbbe88ee1536f17ee7bd462b2ab3772c52"]="onnxtip")
-for onnx_version in ${!version2tag[@]}; do
+version2tag=(5af210ca8a1c73aa6bae8754c9346ec54d0a756e-onnx123
+             bae6333e149a59a3faa9c4d9c44974373dcf5256-onnx130
+             9e55ace55aad1ada27516038dfbdc66a8a0763db-onnx141
+             7d7bc83d29a328233d3e8affa4c4ea8b3e3599ef-onnx150
+             5c51f0dbbe88ee1536f17ee7bd462b2ab3772c52-onnxtip)
+for v2t in ${version2tag[*]}; do
+  onnx_version="$(cut -d'-' -f1<<<${v2t})"
```

Comment:
What are we trying to achieve with this loop? It's somewhat unclear given there are no comments explaining the intention.

I'm not quite understanding why we need to pip install and uninstall so many different versions vs. having a loop to setup /data/onnx (although I'm also not sure what uses that), and a single step to install the latest version.

Can you please clarify?

Reply:
@snnn for this.

Reply:
The key point here is to geneate test data from multiple versions of onnx, test will follow in the end of build pipeline to make sure latest onnxruntime are fine to all of them.

Reply:
As discussed please add comments explaining such things. Unsetting my status in case you need to checkin tomorrow so someone else can sign off.

Reply:
Comments added.